site/src/pages/TemplateSettingsPage/TemplateSettingsForm.tsx

```diff
@@ -12,11 +12,17 @@ import { Stack } from "components/Stack/Stack"
 import { FormikContextType, FormikTouched, useFormik } from "formik"
 import { FC, useRef, useState } from "react"
 import { colors } from "theme/colors"
-import { getFormHelpers, nameValidator, onChangeTrimmed } from "util/formUtils"
+import {
+  getFormHelpers,
+  nameValidator,
+  templateDisplayNameValidator,
+  onChangeTrimmed,
+} from "util/formUtils"
 import * as Yup from "yup"
 
 export const Language = {
   nameLabel: "Name",
+  displayNameLabel: "Display name",
```

Comment:
We're actually moving away from these Language blocks and instead using [react-i18next](https://react.i18next.com/) for translations because it's a cleaner solution. We try to knock them out whenever we come across them in legacy code. Would you mind adding a new translation for your label in `templatePage.json`? There's an [example PR here](https://github.com/coder/coder/pull/3682) that should walk you through how to do it. Let me know if you need any help!

Reply:
Thanks for the suggestion, @Kira-Pilot! https://github.com/coder/coder/pull/5069/commits/6396671dd25c2a282d7359b5d8692e3482b2bcc5